src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -707,8 +710,10 @@ export function annotateGeneticTrackData(
             const newObj = _.clone(object);
             newObj.data = newObj.data.filter(d => {
                 // clear previous annotations
-                delete d.oncoKbOncogenic;
-                delete d.putativeDriver;
+                delete (d as Partial<OncoprinterGeneticTrackDatum_Data>)
```

Comment:
the new typescript has a check which does not allow you to use `delete` on a member which is not potentially undefined in the type